src/ResourceManager/KeyVault/Commands.KeyVault/Models/KeyAttributes.cs

```diff
@@ -65,7 +65,7 @@ public KeyAttributes()
 
         public DateTime? Updated { get; private set; }
 
-        public bool PurgeDisabled { get; private set; }
+        public string RecoveryLevel { get; private set; }
```

Comment:
@dragav removing the `PurgeDisabled` property is a breaking change. Users who run the following will be broken:

```powershell
$recoveredKey = Undo-AzureKeyVaultKeyRemoval -VaultName $vault -Name $keyName
$recoveredKey.Attributes.PurgeDisabled
```

Our recommendation is to keep supporting this property, making it obsolete until the next breaking change release (this fall) when you can then remove it since users will have been warned ahead of time of the change.